com.unity.multiplayer.mlapi/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -77,12 +73,12 @@ public NetworkObject GetLocalPlayerObject()
         /// <returns>The player object with a given clientId or null if one does not exist</returns>
         public NetworkObject GetPlayerNetworkObject(ulong clientId)
         {
-            if (!NetworkManager.ConnectedClients.ContainsKey(clientId))
+            if(NetworkManager.ConnectedClients.TryGetValue(clientId, out NetworkClient networkClient))
```

Comment:
would you mind running `./standards.py --fix` locally to format/fix naming & whitespace issues? :)

Reply:
done